### DISCUSSION TITLE
Is it possible to implement the full system in Foundry?

### DISCUSSION THREAD

**Discussion by chrpow** [Muttley/foundryvtt-shadowdark#564] [Q&A]
I didn't see anything regarding this in the contribution wiki. I'd love to help inputting data for the full system (spells/classes/monsters). Is there a licensing issue that prevents inputting that data into the Foundry system?

**Comment by Muttley** [comment]
We are currently only authorised by The Arcane Library to include content from the Quickstart and Ranger/Bard classes.
